api/v1alpha1/ratelimit_types.go

```diff
@@ -33,6 +33,8 @@ type RateLimitSpec struct {
 	Namespace  string   `json:"namespace"`
 	Seconds    int      `json:"seconds"`
 	Variables  []string `json:"variables"`
+	// +optional
+	LimitadorRef NamespacedName `json:"limitador-ref,omitempty"`
```

Comment:
Not sure we want to continue this path. Maybe better this path https://github.com/Kuadrant/limitador-operator/issues/21 ?